docs/src/cloudformation.rst

```diff
@@ -190,12 +190,12 @@ Intrinsic Functions
 Pseudo Parameters
 -----------------
 
-.. NEEDS SOME INTRO TEXT
+CloudFormation has pseudo parameters that can be used to reference things like the current AWS region, account ID, or stack name. You can add these pseudo parameters in a CDK stack using the `cdk.Aws()` accessor:
 
 .. code-block:: js
 
     import cdk = require('@aws-cdk/cdk');
-    new cdk.AwsRegion()
+    new cdk.Aws(this).region
```

Comment:
Actually, we rather point people to use `Stack.find(this).region` here. Our plan is to make the stack eventually available through `this.node.stack` (or `this.stack`), this will eventually be `this.stack.region`.

Reply:
Sorry for the confusion

Reply:
Isn't `this` already the local stack? 

```js
class GlobalDistribution extends cdk.Stack {
  constructor(parent, id, props) {
    super(parent, id, props);

    new cloudfront.CloudFrontWebDistribution(this, 'MyDistribution', {
     ...
            domainName: props.autocompleteGateway.restApiId + '.execute-api.' + new cdk.Stack.find(this).region + '.amazonaws.com',
      ...
    });
  }
}
```

Could it instead be just `this.region` since `this` should already be the local cdk.Stack with no need for a .find() lookup?

Reply:
You are right, if `this` is already a stack then we can skip the `find`

Reply:
Okay nice. That's actually a better syntax anyway. I'll try it out. Should we update the docs to indicate that this is possible then? And does this pattern apply to all the other pseudo params?

Reply:
Unless you guys say otherwise, I'll add a blurb to the existing "Environments and Authentication" topic, https://docs.aws.amazon.com/CDK/latest/userguide/environments.html, about using this.region IFF this is a stack. I'll also suss out any other instances of cdk.AwsRegion in the guide (I think there is only one in the CFN construct topic).